lib/Common/CommonDefines.h

```diff
@@ -101,7 +101,7 @@
 // ByteCode
 #define VARIABLE_INT_ENCODING 1                     // Byte code serialization variable size int field encoding
 #define BYTECODE_BRANCH_ISLAND                      // Byte code short branch and branch island
-
+#define ENABLE_UNICODE_API 1                        // Enable use of Unicode-related APIs
```

Comment:
Is ```ENABLE_UNICODE_API``` always defined (if we intend to have it always defined, maybe don't need it)? Is there need / support to have it undefined?